packages/common/client-utils/src/cjs/package.json

```diff
@@ -1,7 +0,0 @@
-{
-	"type": "commonjs",
-	"browser": {
-		"./indexNode.js": "./indexBrowser.js",
-		"./indexNode.d.ts": "./indexBrowser.d.ts"
-	}
-}
```

Comment:
@jason-ha - We no longer rely on "browser", so I think we no longer need a custom cjs/package.json?

...but I'm a little confused on why it was there in the first place.  I thought WebPack would use the package.json in the root and that the stub we copy into 'dist/package.json' was only consumed by TypeScript.  Am I mistaken?

Reply:
Webpack was using the nearest package.json for remapping. Without any remapping properties there was no remapping.

Reply:
We no longer rely on remapping for imports after the initial entry point.

This causes a dozen lines or so of code duplication, but I think the simplicity is worth it.